scripts/index.js

```diff
@@ -139,23 +139,61 @@
 
 
 
-// Write a strMultiply function which takes two arguments:
-// str - the string to multiply
-// times - number of times to multiply
-
-function range(min, max) {
-    let arr = [];
-    for (let i = min; i < max; i++) {
-      arr.push(i);
-    }
-    return arr;
-}
-
-function strMultiply(str, times) {
```

Comment:
You solved this one just fine, but the idea was to use the provided range function to do the looping work for you. Can you think of a way you could have done that to include `range(min, max)` and then maybe a `.map`?